core/runtime/types.hpp

```diff
@@ -13,6 +13,11 @@ namespace kagome::runtime {
    * @brief type of wasm memory is 32 bit integer
    */
   using WasmPointer = uint32_t;
+  /**
+   * @brief combination of pointer and size, where less significant part
+   * represents wasm pointer, and most significant represents size
+   */
+  using PointerSize = uint64_t;
```

Comment:
Seems this is a bad name. Think about to change it to WasmBufferRef or somenting else

Reply:
Maybe name is not good, but it was mentioned in form of "pointer-size value" in specs. Your proposal doesn't suit as well, because it is not reference. I was considering FullWasmPointer, but it is ugly. Anyway, if you have a good alternative, i will use it.

Reply:
Let talk about after daily meeting. Remind, pls

Reply:
renamed to WasmSpan